packages/addons/service/rsyslog/source/settings-default.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?> <settings>
+    <category label="1000">
```

Comment:
Remove line #2 all together...? Not sure how '1000' stayed in there. Thought I had updated that.